lib/iris/cube.py

```diff
@@ -2344,20 +2343,14 @@ def _summary_vector_sections_info(self):
         ]
 
         # Ancillary Variables
-        vector_ancillary_variables = [av for av in self.ancillary_variables()]
+        vector_ancillary_variables = [
+            av for av in self.ancillary_variables() if av.shape != (1,)
+        ]
 
-        # Sort scalar coordinates by name.
-        scalar_coords.sort(key=lambda coord: coord.name())
-        # Sort vector coordinates by data dimension and name.
-        vector_dim_coords.sort(
-            key=lambda coord: (self.coord_dims(coord), coord.name())
-        )
-        vector_aux_coords.sort(
-            key=lambda coord: (self.coord_dims(coord), coord.name())
-        )
-        vector_derived_coords.sort(
-            key=lambda coord: (self.coord_dims(coord), coord.name())
-        )
+        # Sort vector coordinates by data dimension.
+        vector_dim_coords.sort(key=self.coord_dims)
+        vector_aux_coords.sort(key=self.coord_dims)
+        vector_derived_coords.sort(key=self.coord_dims)
```

Comment:
I don't actually know about `derived_coords`, but I was fairly certain that you _are_ allowed to have multiple `aux_coords` against a single dimension, so it's still appropriate to keep the second sort by `coord.name()`. Am I wrong?

Reply:
We have now discovered that **all 5** of the components in `section_specs` are intrinsically sorted since that's the way they come out of the `Cube`!